proto/config/patch_envelope.proto

```diff
@@ -35,7 +35,7 @@ message EveOpaqueBase64Data {
 
 message EveOpaqueBinaryBlob {
     // Target field of Volume set to AppCustom
-    Volume opaqueVolume = 1;
+    string opaqueVolume = 1;
```

Comment:
@uncleDecart is this reference a UUID? If so it would make sense to state that in a comment.

Reply:
Yes it is, I'll comment it out

Reply:
Done